roles/config-docker-machine/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Compiled and install docker-machine from source
+  block:
+  - name: config golang
+    include_role:
+      name: config-golang
+  - name: install docker-machine from source
+    shell: |
+      set -ex
+
+      # Install bats
+      git clone https://github.com/sstephenson/bats.git '{{ ansible_user_dir }}/bats'
+      pushd '{{ ansible_user_dir }}/bats'
+      ./install.sh /usr/local
+      popd
+
+      # Install docker
+      apt-get update
+      apt-get install apt-transport-https ca-certificates
+      wget -qO- https://get.docker.com/ | sh
+
+      # NOTE: backport fix of PR 4543 and PR 4545
+      git config --global user.email 'zuul@openlab.com'
+      git config --global user.name 'openlab'
+      git cherry-pick 9ec6729
+      git cherry-pick 7909ee3
```

Comment:
maybe we can switch to use the latest release v0.16.0 and remove this workaround

Reply:
Yes, I consider that too, will do it in separated patch.